pkg/types/cciptypes/models.go

```diff
@@ -0,0 +1,38 @@
+package cciptypes
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
```

Comment:
Not an expert on this but should we be importing go-ethereum here? Seems to defeat the purpose of chain-agnositicism.

Reply:
yes @makramkd is correct.

i thought we had defined agnostic types together @dimkouv ? did we just overlook this or is it more complicated?